src/Appwrite/Platform/Tasks/ScheduleFunctions.php

```
@@ -32,7 +31,7 @@ public static function getCollectionId(): string
         return 'functions';
     }
 
-    protected function enqueueResources(Group $pools, Database $dbForPlatform, callable $getProjectDB): void
+    protected function enqueueResources(Database $dbForPlatform, callable $getProjectDB): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$getProjectDB'. (Unused Code Rules)

(UnusedFormalParameter)